src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/InlineData.cs

```diff
@@ -84,5 +34,18 @@ public static string ResxWithContents (string contents)
 		{
 			return Resx.Replace ("<!--contents-->", contents);
 		}
+
+		static byte[] GetResourceData (string name)
+		{
+			using var s = typeof (InlineData).Assembly.GetManifestResourceStream (name);
+			var data    = new List<byte> ();
+			var buf     = new byte[1024];
+			int r;
+			while ((r = s.Read (buf, 0, buf.Length)) > 0) {
+				for (int i = 0; i < r; ++i)
+					data.Add (buf [i]);
+			}
+			return data.ToArray ();
```

Comment:
Could this do something like this instead?

```csharp
var buf = new byte [s.Length];
s.Read (buf, 0, buf.Length);
return buf;
```
I think a `List<T>` would potentially resize the underlying array a couple times.

Reply:
That *may* work, but I don't know what the semantics of the stream returned by `GetManifestResourceStream()` is.  Is it possible that for "large" files `Stream.Read()` won't read in everything?

Reply:
Maybe this one then?
```csharp
using var s = typeof (InlineData).Assembly.GetManifestResourceStream (name);
using var m = new MemoryStream ();
s.CopyTo (m);
return m.ToArray ();
```

Reply:
We might want to pass stream length to memory stream constructor if we want to avoid resizes during the process.